kubernetes/configmap.yaml

```diff
@@ -0,0 +1,15 @@
+# This Kubernetes configmap file is intended for local environment deployment only.
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: spark-manager-config
+  namespace: cdm-jupyterhub
+data:
+  KUBE_NAMESPACE: "cdm-jupyterhub"
+  SPARK_IMAGE: "spark-standalone:latest" 
+  SPARK_IMAGE_PULL_POLICY: "IfNotPresent"
+  POSTGRES_URL: "postgres:5432"
+  POSTGRES_DB: "hive"
+  POSTGRES_USER: "hive"
+  KBASE_ADMIN_ROLES: "CDM_JUPYTERHUB_ADMIN"
```

Comment:
I've forgotten why this admin role is relevant to this service, can you remind me?

Reply:
yea. We check roles to allow users to create clusters larger than the default one.
https://github.com/kbase/cdm-kube-spark-manager/blob/main/src/routes/clusters.py#L47-L59

Reply:
:+1: